.ci/setup_env_fedora.sh

```diff
@@ -10,17 +10,9 @@ set -e
 cidir=$(dirname "$0")
 source /etc/os-release || source /usr/lib/os-release
 source "${cidir}/lib.sh"
-TEST_CGROUPSV2="${TEST_CGROUPSV2:-false}"
```

Comment:
What's the `TEST_CGROUPSV2` about, @GabyCT?

Reply:
@fidencio it was a flag used for the CI of podman so we can deleted it

Reply:
Currently in kata 2.x, in no one of the CIs this flag has been active, this flag belonged to the podman CI in kata 1.x

Reply:
for podman in kata 1.x, we needed to create a custom image and it was tagged with that flag